packages/components/src/stories/MultiMenuButton.tsx

```diff
@@ -9,8 +9,8 @@ import { storiesOf } from '@storybook/react';
 import { select, text, withKnobs } from '@storybook/addon-knobs';
 
 import './stories.scss';
-import { MenuOption, SubMenu } from '../internal/components/menus/SubMenu';
-import { MultiMenuButton } from '../internal/components/menus/MultiMenuButton';
+import { MenuOption, SubMenu } from '..';
```

Comment:
I take it when you lint these import statements will be merged together.

Reply:
Fixed